rust-runtime/aws-smithy-http/src/byte_stream.rs

```diff
@@ -567,6 +608,11 @@ where
     }
 }
 
+const SIZE_HINT_32_BIT_PANIC_MESSAGE: &str = r#"
```

Comment:
Can we get rid of this now that we don't have a trait mandating that we use `usize`? Is it possible to use `u64` now?

Reply:
I think we can. For our own `ByteStream::size_hint` we can just directly return what's returned by the underlying `http_body::Body::size_hint`. Addressed in 0f5f6be.